docs/adr/0002-monorepo-conversion.md

```diff
@@ -0,0 +1,50 @@
+# 2. Restructure ComfyUI_frontend as a monorepo
```

Comment:
For the record, I'm for this. (And in hindsight should have just done a full code review 😂)